docs/source/tensorbay_cli/getting_start_with_CLI.rst

```diff
@@ -29,21 +29,9 @@ The following is the general format for TBRN:
 
```

Comment:
the file name: `tensorbay_cli/getting_start_with_CLI.rst`
Why `cli` in `tensorbay_cli/` is lowercase, but uppercase in `getting_start_with_CLI.rst`